src/components/button/examples/button-link-new-window/index.njk

```diff
@@ -2,7 +2,11 @@
 {{
     onsButton({
         "text": 'Web chat',
-        "url": '#0',
-        "newWindow": true
+        "url": '/patterns/send-a-web-form/examples/form/',
+        "newWindow": true,
+        "newWindowDescription": 'opens in a new window',
```

Comment:
If we want it to default to say it opens in a new tab don't we want all references to `newWindow` to change to `newTab`?

Reply:
Was just trying to avoid a breaking change tbh

Reply:
Yeah fair enough makes sense to make the change I think, and it's something that is pretty easy to fix. Not a big thing though

Reply:
We have a number of naming inconsistencies we have an epic for, so we'll pick up all those breaking changes at the same time: #1888